tests/integration/features/bootstrap/FeatureContext.php

```diff
@@ -1476,6 +1479,171 @@ public function userSharesRichObjectToRoom($user, $type, $id, $metaData, $identi
 		}
 	}
 
+	/**
+	 * @Then /^user "([^"]*)" creates a poll in room "([^"]*)" with (\d+)(?: \((v1)\))?$/
+	 *
+	 * @param string $user
+	 * @param string $identifier
+	 * @param string $statusCode
+	 * @param string $apiVersion
+	 */
+	public function createPoll(string $user, string $identifier, string $statusCode, string $apiVersion = 'v1', TableNode $formData = null): void {
+		$data = $formData->getRowsHash();
+		$data['options'] = json_decode($data['options'], true);
+		if ($data['resultMode'] === 'public') {
+			$data['resultMode'] = 0;
+		} elseif ($data['resultMode'] === 'hidden') {
+			$data['resultMode'] = 1;
```

Comment:
We can not use autoloading classes of our code inside integration tests.
Also we should not. Because then you can simply change the value of a constant, all tests still pass and all mobile clients break!